sdk/src/azure/iot/az_iot_provisioning_client.c

```diff
@@ -558,3 +563,43 @@ AZ_NODISCARD az_result az_iot_provisioning_client_parse_received_topic_and_paylo
 
   return AZ_OK;
 }
+
+AZ_NODISCARD az_result az_iot_provisioning_client_get_request_payload(
+    az_iot_provisioning_client const* client,
+    az_span custom_payload_property,
+    az_iot_provisioning_client_payload_options const* options,
+    uint8_t* mqtt_payload,
+    size_t mqtt_payload_size,
+    size_t* out_mqtt_payload_length)
+{
+  (void)options;
```

Comment:
nit question: should this happen after the precondition statement?

Reply:
It doesn't seem like even in the repo we're consistent as sometimes the (unreferencedVar); thingy comes up front and others its after preconditions.  In this file FWIW though, it's following pattern.  See `az_iot_provisioning_client_parse_received_topic_and_payload` e.g.